content/zh/2024-10-20-skywalking-go-toolkit-metrics-log/index.md

```diff
@@ -0,0 +1,140 @@
+---
```

Comment:
I can't see your blog at the blog page, https://deploy-preview-748--skywalking-website-preview.netlify.app/blog/

Could you recheck the format?

Reply:
I have only published a blog in the `zh` directory, you can check it out https://deploy-preview-748--skywalking-website-preview.netlify.app/zh/

Reply:
Please put a translation version into EN part. Thanks.

Reply:
done!